hardpy/pytest_hardpy/pytest_wrapper.py

```diff
@@ -144,6 +144,26 @@ def confirm_dialog_box(self, dialog_box_output: str):
             return False
         return True
 
+    def confirm_operator_msg(self, is_msg_visible: bool):
```

Comment:
We need to rename the `confirm_dialog_box` function to `send_data` and use it in this case.

From `api.py` we need to send `str(is_msg_visible)`.